### PR TITLE
Added unit tests for sessions

### PR DESCRIPTION
**Pull request by Drizzt1991**

### CODE REVIEW

**Comment by fafhrd91** [comment]
perfect! thanks.

File: tests/test_client_session.py [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+            }, loop=self.loop)
+        headers = session._prepare_headers([("h1", "h1")])
+        self.assertIsInstance(headers, CIMultiDict)
+        self.assertEqual(headers, CIMultiDict([
+            ("h1", "h1"),
+            ("h2", "header2")
+        ]))
+
+    @unittest.mock.patch("aiohttp.client.ClientSession.request")
+    def test_http_methods(self, patched):
```
Comment (kxepal) [inline tests/test_client_session.py:118]: This test method shouldn't be SO long. Please split it into more trivial cases which would help to see what exactly is broken.

Comment (kxepal) [inline tests/test_client_session.py:118]: The others could be simplified as well. One test method should test one specific case, not all the cases at once.

**Comment by kxepal** [comment]
@fafhrd91 ETOOFAST (:

**Comment by fafhrd91** [comment]
:)
i’m tired with 0.15.2 release

**Pull request merged**